libs/designer/src/lib/core/actions/bjsworkflow/add.ts

```diff
@@ -307,3 +309,14 @@ const getOperationType = (operation: DiscoveryOperation<DiscoveryResultTypes>):
       : Constants.NODE.TYPE.API_CONNECTION
     : operationType;
 };
+
+export const getRootNodeDetails = (workflowState: WorkflowState, connectorId: string): string | undefined => {
+  if (workflowState.graph?.children && connectorId.indexOf(Constants.INVOKER_CONNECTION.DATAVERSE_CONNECTOR_ID) > -1) {
```

Comment:
This is super specific to a particular connector, something we want to avoid. Is there a way we can generalize this behavior. Also can you explain why it needs this special treatment at all?

Reply:
The invoker connection is very specific to Dataverse actions. It should only be shown as a setting on Dataverse actions if the trigger being used is a Dataverse CUD trigger and so that's why it needs the specific checks. I'm open to taking suggestions if there's a way to generalize this but afaik this is very specific to DV actions.